dependent/Turcica.csl

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Revue TURCICA </title>
+    <title-short>TURCICA </title-short>
+    <id>http://www.zotero.org/styles/revue-turcica</id>
+    <link rel="self" href="http://www.zotero.org/styles/revue-turcica"/>
+    <contributor>
+      <name>Hélène Morlier </name>
+      <email>helene.morlier@college-de-france.fr</email>
+    </contributor>
+    <contributor>
+      <name>Lucas Equinoxe</name>
+      <email>lucas.equinoxe@college-de-france.fr</email>
+    </contributor>
+    <updated>2022-09-23T07:46:36+00:00</updated>
```

Comment:
Please add the ISSNs as explained here: https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md#csl-style-repository-requirements